feature-libs/order/core/store/reducers/index.ts

```diff
@@ -25,29 +25,29 @@ import * as fromUserOrdersReducer from './orders.reducer';
 import * as fromReplenishmentOrderDetailsReducer from './replenishment-order-details.reducer';
 import * as fromUserReplenishmentOrdersReducer from './replenishment-orders.reducer';
 
-export function getReducers(): ActionReducerMap<Partial<OrderState>> {
+export function getReducers(): ActionReducerMap<OrderState, any> {
```

Comment:
Was there an alternative to use any in these types?

Reply:
It is something extends Action.